radixdlt-core/docs/api-info.md

```diff
@@ -0,0 +1,184 @@
+| radix.transactionHistory | Transaction history for specified account |
+| radix.lookupTransaction | Lookup specified transaction |
+| radix.statusOfTransaction | Status of specified transaction |
+| radix.buildTransaction | Assemble transaction given specified actions and optional message |
+| radix.finalizeTransaction | Send signed transaction and receive transaction ID|
+| radix.submitTransaction | Submit signed transaction for processing by network|
+| radix.validators | Get list of validators |
+| radix.lookupValidator | Get information about specified validator |
+| radix.networkId | Get network ID |
+| radix.networkTransactionThroughput | Get number of transactions per second |
+| radix.networkTransactionDemand | Get average number of transactions waiting for processing in mempool |
+
+### Configuration Options
+
+Following configuration options control which APIs are enabled at the node:
+
+| Configuration Option | Description |
+| --- | --- |
+| client_api.enable | Enables JSON-RPC APIs|
+| universe_api.enable | Enables `/universe.json` API |
+
+</details>
+
+## New Organization of APIs
+
+ The general approach to the API organization is based on the following considerations:
+- Most methods are spread across five endpoints:
+  - `/archive` (formerly `/rpc`) - read-only methods, requiring an archive node
+  - `/construction` - methods which support building and submitting transactions, which can be enabled by either archive or full nodes
+  - `/account` supports methods to fetch your associated account info, and a one-step method to build, sign, and submit a transaction
+  - `/validation` - read-only methods which provide same information as available today via all `/node/*` endpoints
+  - `/system` - read-only methods which provide same information as available today via all `/system/*` endpoints
+    
+- The `/system`, `/account` and `/validator` endpoints are expected to be protected by firewall and/or require authentication/etc. 
+  (similar requirements/setup as we have today)
+
+### REST APIs
+
+Majority of the REST APIs are replaced with JSON-RPC counterparts. Remaining and new REST endpoints:
+
+| Path | Method | Description |
+| --- | --- | --- |
+| /health | GET | Returns node status (see below) |
+| /version | GET | Returns detailed information about software version and build info |
+| /universe.json | GET | Get Radix Universe. Used during setup and configuration of the node but if possible looking to remove |
+
+Node health status has following format: `{"status" : "<status>" }`, where `<status>` is one of the following:                             
+ - BOOTING - node is booting and not ready to accept requests
+ - SYNCING - node is catching up the network
+ - UP - node is in sync with consensus
+ - STALLED - node is out of sync and not trying to sync with network, but network is still available.
+ - NETWORK_HALTED - node is out of sync and does not get updates from network (for example, connection to network is lost).
```

Comment:
Suggest renaming this to `OUT_OF_SYNC`. It's a node issue rather than network issue.

Reply:
makes sense